### PR TITLE
Add runtime metric for CPU usage in percent

### PR DESCRIPTION
**Pull request by kevingosse**

### CODE REVIEW

File: src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs [hunk 1]
```diff
@@ -103,18 +103,24 @@ private void PushEvents()
...
-                    // Get the CPU time per second
-                    var userCpu = (newUserCpu - _previousUserCpu).TotalMilliseconds / (_delay / 1000.0);
-                    var systemCpu = (newSystemCpu - _previousSystemCpu).TotalMilliseconds / (_delay / 1000.0);
+                    var userCpu = newUserCpu - _previousUserCpu;
+                    var systemCpu = newSystemCpu - _previousSystemCpu;
 
                     _previousUserCpu = newUserCpu;
                     _previousSystemCpu = newSystemCpu;
 
+                    var maximumCpu = Environment.ProcessorCount * _delay;
```
Comment (zacharycmontoya) [inline src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs:112]: I'm not sure if you've encountered this thread before, but maybe it's worth adding a comment that links to this issue https://github.com/dotnet/runtime/issues/622 , which outlines a regression when using `Environment.ProcessorCount` in containerized environments misbehave in .NET Core 2.x and 3.0?

Comment (zacharycmontoya) [reply src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs:112]: I see you commented in a related CPU quota issue 😄  https://github.com/dotnet/runtime/issues/13224

Comment (kevingosse) [reply src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs:112]: I don't mind mentioning it. But to be honest I lost track of what versions have what behavior :D All I know is the latest versions report the number of cpus attributed to the container, which is what we want

Comment (kevingosse) [reply src/Datadog.Trace/RuntimeMetrics/RuntimeMetricsWriter.cs:114]: Added in https://github.com/DataDog/dd-trace-dotnet/pull/1064/commits/3c160d2854796b168e7b29b6ad428522438fc143

**Review by zacharycmontoya — APPROVED** [review]

**Review by tonyredondo — APPROVED** [review]

**Pull request merged**